test/unit/utils.test.ts

```diff
@@ -642,4 +645,133 @@ describe('driver utils', function () {
       });
     });
   });
+
+  describe('maybeCallback()', () => {
+    it('should accept to two arguments', () => {
```

Comment:
"should accept two arguments"? or "up to two"?

Reply:
Sure I can correct this wording, however we should never call it without 2 arguments though, its just something we can't verify since there's no difference between a variable = undefined vs omitting the argument. We added a TS overload to allow null to kind of hint at this but we just have to rely on testing / double checking the code to make sure we pass through the callback variable if there is one.

Reply:
fixed!